x/datapool/genesis.go

```diff
@@ -12,6 +12,10 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	k.SetParams(ctx, genState.Params)
 	k.SetPoolNumber(ctx, genState.NextPoolNumber)
 
+	if genState.NftContractAddress != nil {
+		k.SetNFTContractAddress(ctx, genState.NftContractAddress)
+	}
```

Comment:
I think it's okay to remove this `if` statement, even though the `DefaultGenesis.NftContractAddress` is `nil`: https://github.com/medibloc/panacea-core/pull/295/files#diff-1b224410c503ee7f657889a5370b3ff80d2396445a85534d2e4a544bbe357aecR12.

Reply:
Ah, I understood why you added this `if` statement. 
https://github.com/medibloc/panacea-core/blob/722287ea7d0b093a1a3517a54310791f5bdab032/x/datapool/keeper/pool.go#L253

Okay then, let's go as it is.